Michael-Walter.txt

```diff
@@ -0,0 +1,149 @@
+https://codepen.io/MPAW/pen/WNbbjxz?editors=1100
+
+    1. What is Semantic HTML? 
+	a. a logical use of meaningful html elements. 
+
+    2. What is HTML used for? 
+	a. html is used to structure and present content.
+
+    3. What is an attribute and where do we put it? 
+	a. attributes can be placed inline, within the head, or within a css document.
+
+    4. What is the h1 tag used for? How many times should I use it on a page?
+	a. It's used as a main header and should only be used once on a page.
+
+    5. Name two tags that have required attributes
+	a. img tag, requires a src.
+	b. input tag, requires a type
+
+    6. What do we put in the head of our HTML document? 
+	a. title, possible comment, link to css doc, link to small js script	
+
+    7. What is an id? 
+	a. It's a "silver bullet" used mostly for anchors
+
+    8. What elements can I add an id to? 
+	a. it can technically be added to all, but in best practices it should apply to containers, or styling elements.
+ 	
+    9. How many times can I use the same id on a page? 
+	a. a "silver bullet" can only be used once.
+
+    10. What is a class? 
+	a. a multi-element targeting identifier. 
+
+    11. What elements can I add a class to? 
+	a. all elements can be classy, but styling as well as containers do it best. 
+
+    12. How many times can I use the same class on a page? 
+	a. it can be used as many times as you want.
+
+    13. How do I get my link to open in a new tab?
+	a. by adding, targer="_blank" to your anchor element.
+
+    14. What is the alt attribute used for? 
+	a. to show text if the image doesn't load, and for screen readers.
+
+    15. How do I reference an id?
+	a. #id
+
+    16. What is the difference between a section and a div
+	a. the major difference is that crawlers, RSS feeders, and some AI's that are being used for data aggregation are more commonly trained to grab data by identifying sections of a page, and not the numerous divs.
+	b. once is a container and not really semantic html, whereas section is.
+	c. section divides major blocks of the page, and div contains or divides minor componenets.
+
+    17. What is CSS used for? 
+	a. for styling HTML and minorly improving the functionality of the page, inproving the presentation.
+
+    18. How to we select an element? Example - every h2 on the page
+	a. h2 {...}
+
+    19. What is the difference between a class and an id? - Give me an example of when I might use each one
+	a. classes have unlimited uses, and ID's have a single use.
+	b. classes are more commonly used to target elements for styling, especially when a style may occur more than once on a page. ID's are more commonly used for anchoring, but can be used for single target styling however seldom.
+
+    20. How do we select classes in CSS?
+	a.  .class {...}
+
+    21. How do we select a p element with a single class of “human””?
+	a. p .human {...}
+
```

Comment:
Remember that the p element you are targeting has the class of `human`. What your code is saying here is `find a p tag and then look inside that p tag for any element that has the classname human` But what you want to do is to find any p tag that HAS the classname of human. the space is what is throwing it off. `p.human` is what you would do if you were wanting to select the p tag with class of human